src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/executors/spark/SparkEmrClusterServiceImpl.scala

```diff
@@ -58,6 +58,11 @@ class SparkEmrClusterServiceImpl @Inject()(sparkConfig: SparkConfig, emrClusterP
     }
   }
 
+  override def handleMissingYarnStatusForJobStatusSubmitting(jobInstance: JobInstance, updateJob: JobInstance => Future[Unit])
+    (implicit executionContext: ExecutionContext): Future[Unit] = Future {
+    // do nothing
```

Comment:
If something goes wrong, job will be never executed and end in submitting status.